cutlass/nvf_cutlass.h

```diff
@@ -19,4 +19,17 @@ torch::Tensor nvfp4_scaled_mm(
     const torch::Tensor& alpha,
     at::ScalarType out_dtype);
 
+void nvfp4_scaled_grouped_mm(
+    torch::Tensor& output,
```

Comment:
@jjsjann123 Do you want to return `torch::Tensor output` from this function?

Reply:
it's fine to keep it as-is. Not a big issue on my side.